app/src/main/java/univ/pr/nj/keewitz/ReportAnomalyActivity.java

```diff
@@ -0,0 +1,114 @@
+package univ.pr.nj.keewitz;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Environment;
+import android.os.StrictMode;
+import android.provider.MediaStore;
+import android.support.v4.app.ActivityCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.Spinner;
+import android.widget.Toast;
+
+import java.io.File;
+import univ.pr.nj.keewitz.utils.FirebaseUtils;
+
+
+public class ReportAnomalyActivity extends AppCompatActivity implements AdapterView.OnItemSelectedListener {
+
+    private Uri imageUri;
```

Comment:
Can you set this 3 fields in a `model/Anomaly.java` to have a clear `DAO` class.